src/styles/global/typography.scss

```diff
@@ -130,9 +130,12 @@ h6 {
 }
 
 .site-footer {
-    color: $ebb;
+    color: $alabaster;
     a {
-        color: $ebb;
+        color: $alabaster;
+    }
+    a:hover {
+        color: $frenchGray;
     }
```

Comment:
Yep - we'll be pulling it in from Pugin as part of the language agnostic component library before release.